hikari/impl/shard.py

```diff
@@ -87,6 +88,20 @@
 # Rate-limit for chunking requests (used to prevent saturating the entire
 # ratelimit window).
 _CHUNKING_RATELIMIT: typing.Final[typing.Tuple[float, int]] = (60.0, 60)
+# Supported gateway version
+_VERSION: int = 6
```

Comment:
This gets defined here but never used. Is that intentional?

Reply:
nope, fixed.